Train.json

```
@@ -560,6 +575,25 @@
 				{"name": "passengers", "value": 121}
 			]
 		},
+		{
+			"id": 756,
+			"group": 1,
+			"name": "Dreiachser 2. Klasse",
+			"shortcut": "B3yg54",
+			"speed": 100,
+			"weight": 18,
+			"force": 0,
+			"length": 13,
+			"drive": 0,
+			"reliability": 0.7,
+			"cost": 100000,
+			"maxConnectedUnits": 0,
+			"operationCosts": 10,
+			"exchangeTime": 100,
+			"capacity": [
+				{"name": "passengers", "value": 62}
+			]
+
```

Der linter meint, du solltest das object noch abschließen.

```suggestion
		},
```